userspace/core/ls.c

```diff
@@ -84,6 +85,9 @@ static void print_entry(struct tfile * file, int colwidth) {
 	if (S_ISDIR(file->statbuf.st_mode)) {
 		/* Directory */
 		ansi_color_str = DIR_COLOR;
+	} else if (S_ISLNK(file->statbuf.st_mode)) {
```

Comment:
There's some dumb duplication of the coloring between short and long output modes in `ls` and you missed the other one. Should probably deduplicate it at some point...